Simplified/NYPLBookCell.m

```diff
@@ -51,6 +51,8 @@ void NYPLBookCellRegisterClassesForCollectionView(UICollectionView *const collec
   NYPLBookState const state = [[NYPLBookRegistry sharedRegistry]
                                stateForIdentifier:book.identifier];
   
+    // So much repetition here. Needs fixing.
```

Comment:
yes indeed -- there are a few places in the code like this unfortunately. We should file tech debt tickets for these refactors.

Reply:
Agreed. Let's fix these in a separate ticket.

Reply:
we also have a budding effort to restyle some of these screens, so I think we will do that then.